process-services/1.11/config/index.md

```diff
@@ -0,0 +1,180 @@
+
+    For other servers there will be other ways of setting environment/JVM variables. These values can be read as JVM parameters, environment variables or as property file entries (though you would not want to put the secret encryption password in a property file). Therefore, with WebSphere they could set using JVM parameter config [http://www-01.ibm.com/support/docview.wss?uid=swg21417365](http://www-01.ibm.com/support/docview.wss?uid=swg21417365){:target="_blank"} or environment variable config [https://www.ibm.com/support/knowledgecenter/en/SSAW57_8.5.5/com.ibm.websphere.nd.doc/ae/welcvariables.html.](https://www.ibm.com/support/knowledgecenter/en/SSAW57_8.5.5/com.ibm.websphere.nd.doc/ae/welcvariables.html){:target="_blank"}.
+
+8. Start the application.
+
+    The application should now start as normal. If it doesn’t, try without the encrypted values and without the encryption parameters to determine whether the problem is related to the encryption setup. Check that you are able to encrypt and decrypt with Jasypt to rule out any issues due to copy-paste errors. 
+
+9. Logging.
+
+    Some property values (though not sensitive ones) are logged by Alfresco applications if the log level is set high. If you want to restrict this then reduce the log level in`log4j.properties`
+
+## CORS
+
+To enable Cross Origin Resource Sharing (CORS) in Process Services, set the `cors.enabled` property to true in the
+`activiti-app.properties` file.
+
+>**Note:** This feature is only supported on Tomcat application server.
+
+```text
+# CORS CONFIGURATION
+#
+cors.enabled=true
+```
+
+When CORS is enabled, CORS requests can be made to all endpoints under `/activiti-app/api`.
+
+Also, some additional properties are made available which can be configured to further fine tune CORS. This will make CORS available only to certain origins or to restrict the valid HTTP methods that can be used and headers that can be sent with CORS-enabled requests.
+
+```text
+cors.enabled=false
+cors.allowed.origins=*
+cors.allowed.methods=GET,POST,HEAD,OPTIONS,PUT,DELETE
+cors.allowed.headers=Authorization,Content-Type,Cache-Control,X-Requested-With,accept,Origin,Access-Control-Request-Method,Access-Control-Request-Headers,X-CSRF-Token
+cors.exposed.headers=Access-Control-Allow-Origin,Access-Control-Allow-Credentials
+cors.support.credentials=truecors.preflight.maxage=10
+```
+
+|Property|Description|
+|--------|-----------|
+|cors.allowed.origins|Specifies the hosts allowed in cross origin requests. By default, the value is set to `*`, which permits clients hosted on any server to access the resources. Alternatively, you can specify a host, for example, `http://www.example.org:8080`, which will only allow requests from this host.<br><br>Multiple entries or wildcards are not allowed for this setting.<br><br>In general, it is recommended to restrict `{{allowedOrigins}}` to only allow origins within your organization to make requests.|
```

Comment:
`*` can no longer used as the value for this property. By default it's not set. A new property `cors.allowed.origin.patterns` should be used in case `*` needs to be used.

Reply:
cors.allowed.origins does not start accept * it has to be a specific name, e.g. alfresco.com

cors.allowed.origin.patterns if you want to use a * then you can access the 
descriptions can be the same emphasize second one can be * first one has to be specific name.

Reply:
Done